web-platform/website/src/components/pages/profile/__tests__/ProfileButtons.tests.tsx

```diff
@@ -55,9 +55,7 @@ describe("ProfileButtons tests", () => {
     expect(push).not.toHaveBeenCalled();
 
     await userEvent.click(cancelButton);
-
-    expect(push).toHaveBeenCalledWith("/");
-
+  
```

Comment:
We should verify push after awaiting on setTimeout. Maybe we can mock out setTimeout and make it sequential. Same for changes below.

Reply:
I tried mocking setTimeout using jest.useFakeTimers() and jest.setTimeout(time) but neither worked so I just used the setTimeout fn and the test passed. Let me know if you have issues with this.